src/chainbridgeConfig.ts

```diff
@@ -59,3 +65,19 @@ export type UIConfig = { wrapTokenPage: boolean };
 
 export const chainbridgeConfig: ChainbridgeConfig =
   window.__RUNTIME_CONFIG__.CHAINBRIDGE;
+
+export const getСhainConfig = (chainId: number) => {
```

Comment:
1. You've changed the logic to identify the network. Why?
2. Rename the method. It relates to Substrate chain only

Reply:
1. Last time we talked that `!==` operator is not straightforward.
2. It's relevant for all chain types in our config file. Why it's relevant for Substrate chain only?